tests/test_squad.py

```diff
@@ -0,0 +1,295 @@
+# Copyright (c) 2019 NVIDIA Corporation
+import unittest
+import os
+import sys
+import json
+
+import shutil
```

Comment:
Please group this with the rest of the Standard library imports.

Reply:
I changed the source branch accordingly